src/components/menu/__internal__/submenu/submenu.component.js

```diff
@@ -399,6 +411,8 @@ Submenu.propTypes = {
   onSubmenuOpen: PropTypes.func,
   /** Callback triggered when submenu closes. Only valid with submenu prop */
   onSubmenuClose: PropTypes.func,
+  /** @private @ignore */
```

Comment:
This isn't really needed for an `internal` component but it doesn't need to be removed now it's been added

Reply:
Fair enough - not sure why I didn't notice it was internal! I'll leave as is for now but if someone else requests changes I'll remove that at the same time.

Reply:
now removed and replaced with a proper comment as Jamie requested changes